stripe/src/main/java/com/stripe/android/view/ShippingFlowConfig.java

```diff
@@ -0,0 +1,85 @@
+package com.stripe.android.view;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.stripe.android.model.Address;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class ShippingFlowConfig implements Parcelable {
+
+    private List<String> mHiddenAddressFields;
+    private List<String> mOptionalAddressFields;
+    private Address mPrepopulatedAddress;
+    private boolean mHideAddressScreen;
+    private boolean mHideShippingScreen;
+
+    public ShippingFlowConfig(List<String> hiddenAddressFields, List<String> optionalAddressFields, Address prepopulatedAddress, boolean hideAddressScreen, boolean hideShippingScreen) {
```

Comment:
I think I would prefer to `@NonNull` all the things, and have the lists instantiated as empty array lists in the constructor in that previous class if they haven't already been initialized.